clio-server/src/test/scala/org/broadinstitute/clio/server/service/ServerServiceSpec.scala

```diff
@@ -170,7 +168,19 @@ class ServerServiceSpec
     } yield {
       val upsertedDocs = searchDAO.updateCalls
         .flatMap(_._1)
-      upsertedDocs should contain theSameElementsAs initStoredDocuments
+
+      upsertedDocs.take(initInSearch) should contain theSameElementsAs initStoredDocuments
+        .take(initInSearch)
+
+      upsertedDocs.map(_.mapObject(_.filter {
+        case (k, _) => k != "mock_default_field"
+      })) should contain theSameElementsAs initStoredDocuments
+
+      upsertedDocs
+        .drop(initInSearch)
+        .forall(
+          js => js.asObject.getOrElse(JsonObject.empty).contains("mock_default_field")
+        ) should be(true)
```

Comment:
Same comment as in the integration tests, I think it'd be better to instead inject `mock_default_field` into the expected documents and then use one `should contain`.